packaging/dependency/lxd.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package dependency
+
+import (
+	"github.com/juju/errors"
+	"github.com/juju/juju/packaging"
+)
+
+const blankSeries = ""
+
+// LXD returns a dependency instance for installing lxd support
+func LXD() packaging.Dependency {
+	return lxdDependency{}
+}
+
+type lxdDependency struct{}
+
+// PackageList implements packaging.Dependency.
+func (lxdDependency) PackageList(series string) ([]packaging.Package, error) {
+	var pkg packaging.Package
+
+	switch series {
+	case "centos7", "opensuseleap", "precise":
```

Comment:
I'm unsure if this should be done here, esp. considering if we have `os/series`?

Reply:
Essentially I'm unsure if "precise" should be here!